pkg/kubernetes/kubectl.go

```diff
@@ -0,0 +1,48 @@
+package kubernetes
+
+import (
+	"fmt"
+	"github.com/alexellis/go-execute/pkg/v1"
+	"strings"
+)
+
+func Kubectl(parts ...string) error {
+	task := execute.ExecTask{
+		Command:     "Kubectl",
+		Args:        parts,
+		StreamStdio: true,
+	}
+
+	res, err := task.Execute()
+
+	if err != nil {
+		return err
+	}
+
+	if res.ExitCode != 0 {
+		return fmt.Errorf("Kubectl exit code %d, stderr: %s",
+			res.ExitCode,
+			res.Stderr)
+	}
+	return nil
+}
+
+func GetNodeArchitecture() string {
+	res, _ := KubectlTask("get", "nodes", `--output`, `jsonpath={range $.items[0]}{.status.nodeInfo.architecture}`)
+
+	arch := strings.TrimSpace(string(res.Stdout))
+
+	return arch
+}
+
+func KubectlTask(parts ...string) (execute.ExecResult, error) {
+	task := execute.ExecTask{
+		Command:     "Kubectl",
```

Comment:
Again this breaks on Linux, perhaps you tested on MacOS where some commands can run case insensitive?